BMP.md

```diff
@@ -0,0 +1,245 @@
+# BGP Monitoring Protocol (BMP)
+
+https://tools.ietf.org/html/rfc7854
+
+### BMP message types and GoBMP:
+GoBMP will not need to collect and parse the following BMP message types:
+```
+collector
+router
+bmp_stat
+base attribute
+```
+
+GoBMP will collect and parse the following BMP message types (details below):
+```
+peer
+unicast_prefix 
+ls_node
+ls_link
+ls_prefix
+l3vpn
+evpn
+```
+With much gratitude we are using OpenBMP's message parsing as a template:
+
+https://www.snas.io/docs/
+
+#### BMP peer message:
+```
+1: (action): up/down
```

Comment:
Please confirm if `action` field is a string with actual strings "up" or "down" or it is numerical enumeration where for example 0 means up and 1 means down.